packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -306,10 +315,11 @@ export class GCFBootstrapper {
     }
   }
 
-  async getInstallationToken(installationId: number) {
-    return await this.probot!.app!.getInstallationAccessToken({
-      installationId,
-    });
+  // TODO: How do we still get access to this installation token?
+  async getAuthenticatedOctokit(installationId: number): Promise<Octokit> {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const app = (this.probot as any).apps[0] as Application;
```

Comment:
@gr2m I think this is what we talked about earlier today, that can be used to give us an Octokit instance with permission to call `octokit.apps.listReposAccessibleToInstallation`?

Reply:
yes, sorry for the need of this ugly hack, I hope make it obsolete soon

Reply:
Just so we don't lose track, @bcoe could I trouble you to file an upstream bug and leave it in a comment here so we can come clean up?